pkg/virt-controller/watch/migration_test.go

```diff
@@ -60,16 +60,16 @@ import (
 
 var _ = Describe("Migration watcher", func() {
 
-	var ctrl *gomock.Controller
-	var controller *MigrationController
-	var recorder *record.FakeRecorder
-	var mockQueue *testutils.MockWorkQueue
-	var virtClient *kubecli.MockKubevirtClient
-	var virtClientset *kubevirtfake.Clientset
-	var kubeClient *fake.Clientset
-	var networkClient *fakenetworkclient.Clientset
-	var qemuGid int64 = 107
-	var namespace k8sv1.Namespace
+	var (
+		controller    *MigrationController
+		recorder      *record.FakeRecorder
+		mockQueue     *testutils.MockWorkQueue
+		virtClientset *kubevirtfake.Clientset
+		kubeClient    *fake.Clientset
+		networkClient *fakenetworkclient.Clientset
+		namespace     k8sv1.Namespace
```

Comment:
Well, here namespace is used in a complex way.
It is used to create a proper migration policy during some tests, and some labels are added runtime to the namespace.
I agree that this can be refactored but I would leave it for future improvements since it is out of the scope of the PR. WDYT?